README.md

```diff
@@ -87,28 +83,25 @@ environment = var.environment
 label_order = var.label_order
 
 google_kms_key_ring_enabled               = true
-google_kms_crypto_key_enabled             = true
+google_kms_crypto_key_enabled             = true    # when prevent_destroy required true use this key (long-lived key)
+google_kms_crypto_key_ephemeral_enabled   = false   # when prevent_destroy required false use this key (short-lived key)
 google_kms_crypto_key_iam_binding_enabled = true
 project_id                                = var.gcp_project_id
 location                                  = var.location
 keys                                      = ["dev"]
-prevent_destroy                           = false
 service_accounts                          = ["serviceAccount:service-xxxxxxxxxxxxxxx.gserviceaccount.com"]
 role                                      = "roles/cloudkms.cryptoKeyEncrypterDecrypter"
 }
 ```
 
 
-
-
-
-
 ## Inputs
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
 | environment | Environment name | `string` | `"dev"` | no |
-| google\_kms\_crypto\_key\_enabled | (Optional) Whether or not to create resources within the module. | `bool` | `true` | no |
+| google\_kms\_crypto\_key\_enabled | create an ephemeral Google Cloud KMS crypto key within the module. Set to true to include it. | `bool` | `true` | no |
+| google\_kms\_crypto\_key\_ephemeral\_enabled | create an ephemeral Google Cloud KMS crypto key within the module. Set to false to exclude it." | `bool` | `true` | no |
```

Comment:
same for inputs it will generate using github action

Reply:
please verify once